src/content/concepts/why-webpack

```diff
@@ -0,0 +1,55 @@
+---
+title: Why Webpack
+sort: 7
+contributors:
+- debs_obrien
+---
+
+##Why webpack?
+
+To understand why you should use webpack you first have to understand the origins of how we use JavaScript on the web. 
+
+JavaScript is just a script and there are 2 ways to use it in the browser, loaded as a script tag or written directly inside your html. The problems that this causes is that it doesn't scale. You either have too many scripts that you try to load from your script tags in the html and this causes a huge performance problem. There is only a certain amount of concurrent requests that can fetch data at a single time so breaking your application into a hundred JavaScript files and loading them in the browser is a huge performance problem. One big JS file results in unmaintainable scripts that causes problems in scope, size, readability, fragility and monolith files.
```

Comment:
`The problems that this causes is that it doesn't scale.` -> `, this is a problem, it doesn't scale.`

Reply:
Please uppercase `html`: `HTML`.

Reply:
done